Add slovenian translation

LinuxHeki opened the pull request.

annemedia commented:
Hello Heki, first, thank you for delivering the translation, I appreciate your efforts.

Regrettably, I'm reluctant to accept it because I can see it's a near (98-99%) match with the result of Google Translate, and there are quite a few English phrases that are English-specific yet the translation goes word by word as seen on GT. I suspect that the context does not represent accurately some of the meanings in consideration of language differences and nuances and that a native Slovenian person will find some of the narrative mechanical and awkward.

Granted, I'm not a native Slovenian speaker so if you feel the current translation is 100% accurate feel free to defend it. Alternatively, if you feel my feedback is justified, can you please give it a thorough review and deliver a revision?

Thanks again.

LinuxHeki commented:
Hi! It's near match with Google Translate, because it is Google Translate. I used it to make the process faster, but most of the strings (especially long ones) I changed, because they weren't grammatically correct or there was better translation for a word. If you translate fixed strings back to English, you will get the original string most of the time.

Due to Slovenian being a small language, we don't have very good translation for some cases, for example peer-to-peer, startups, FUNDRAISER (I hate that word, it's in every third string), secret view key, etc., so I sticked with the translation that Google gave me, which was not bad.

I looked through the translation for English-specific phrases and I found only one (on the go). I think in Slovenian it's not wrong if you translate word by word, but I also found a better word for that phrase. While looking I've also found some typos and one untranslated string, so I fixed that.

I am a native Slovenian speaker and I can confirm it's accurate. Maybe it's not a 100% accurate, but it's defenetly 99% :).

annemedia commented:
We have specifically warned on our GitHub listing not to deliver Google translations because they are not accurate. The very point of any translation is not to make it literal but native, it's not supposed to translate back to the original string. The fact you suggest otherwise is a big red flag, and the fact you suggest that Slovenian is a small language is a weak excuse and it raises an alarm. Slovenian is a Slavic language. Slavic languages are a relatively homogeneous family and all are exceptionally rich.

For that reason, we opted to reject the translation. Since your ask has been funded, and you've failed to deliver a reasonable standard, I can only suggest that you determine the right course of action.
